perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -3820,28 +3820,20 @@ private void storeApplicationAttributes(Application app) throws UserNotExistsExc
 				// if attribute exists
 				if (a != null) {
 					if (a.getType().equalsIgnoreCase(LinkedHashMap.class.getName())) {
-						// FIXME do not set hash map attributes - not supported in GUI and registrar
+
+						// we expect that map contains string keys and values
+						LinkedHashMap<String, String> value = a.valueAsMap();
+						value = handleMapValue(value, newValue);
+
+						a.setValue(value);
+						attributes.add(a);
 						continue;
```

Comment:
Not your change, but I think `continue` is not necessary here and in second branch as there is no more code after this whole `if` section.

Reply:
Redundant continue was removed.